gui-apps/organicmaps/organicmaps-9999.ebuild

```
@@ -1,78 +1,102 @@
-# Copyright 1999-2023 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{11..12} )
-inherit git-r3 python-r1 xdg cmake
+PYTHON_COMPAT=( python3_{11..14} )
+inherit git-r3 python-any-r1 xdg cmake
 EGIT_REPO_URI="https://github.com/${PN}/${PN}.git"
-# this URL is to make the tests compile since organicmaps usually dynamically clones the repo
-# maybe a better way would be to skip the test
+
 EGIT_WORLD_FEED_REPO_URI="https://github.com/${PN}/world_feed_integration_tests_data.git"
+
 # organicmaps gets more and more system libraries, we use as many
 # as currently possible, use submodules for the rest
 EGIT_SUBMODULES=(
-	3party/harfbuzz/harfbuzz
 	3party/fast_double_parser
+	3party/fast_obj
+	3party/glaze
+	3party/glfw
+	3party/harfbuzz/harfbuzz
+	3party/imgui/imgui
 	3party/just_gtfs
 	3party/protobuf/protobuf # wait for https://github.com/organicmaps/organicmaps/pull/6310
-	3party/fast_obj
 )
 
 DESCRIPTION="Offline maps and navigation using OpenStreetMap data"
 HOMEPAGE="https://organicmaps.app"
 
 LICENSE="Apache-2.0"
+
 SLOT="0"
 
-REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	dev-util/vulkan-headers
+	${PYTHON_DEPS}
+	test? ( dev-cpp/gtest )
+"
 
 # depend on sys-libs/zlib[minizip] when it is not pulled in as subproject anymore
 RDEPEND="
+	dev-cpp/fast_double_parser
 	dev-cpp/gflags
 	dev-db/sqlite
-	dev-lang/python
 	dev-libs/boost
 	dev-libs/icu
 	dev-libs/jansson
```

```suggestion
	dev-libs/jansson
	dev-libs/pugixml
```
dev-libs/pugixml is missing.
https://github.com/organicmaps/organicmaps/blob/6baea54ad361615cf679d020ddffdc1e2d73a78f/CMakeLists.txt#L188